src/html.css

```
@@ -0,0 +1,14 @@
+html {
+  font-family: "Roboto", Arial, sans-serif;
+}
+
+* {
+  box-sizing: border-box;
+  margin: 0;
```

```suggestion
* {
  box-sizing: border-box;
```
Don't reset margins using * selector. Better specify the exact tags you want to reset